css/style.css

```diff
@@ -298,7 +298,19 @@ button:active {
   color: #f4f1f0;
 }
 
-.recent-works button:not(.first_button){
+.work-card > button {
+  width: 100%;
+  cursor: pointer;
+}
+
+.about-text > button {
+  width: 161px;
+  height: 48px;
+  top: 395px;
+  left: 24px;
+}
```

Comment:
- [ ] as the previous reviewer requested kindly add margins (spaces) between all of your project's cards like shown in Figma

figma design : 
![image](https://user-images.githubusercontent.com/89282221/191565802-af7714f8-657d-4749-b577-014b0651eb29.png)

Reply:
But how ???
I didn't get sorry/
tell me clearly, I think already is good.

Reply:
@ZuhRa-HashiMi try to tag my name always so I can see your comment, I saw it by luck.
you can add a margin-top for each card and it will be fixed easy, the card should have empty space between one and another

Reply:
@ichala Thanks so much.
please see my repo if you accept then I will submit it for code reviewer, because this is my last chance and I don't want to become repeat.

thanks so much.
https://github.com/ZuhRa-HashiMi/portfolio_mobile/pull/2

Reply:
@ZuhRa-HashiMi i will see it rn

Reply:
@ichala Thanks so much
ok I am going to send.